src/ImathTest/testExtractSHRT.cpp

```diff
@@ -32,7 +32,7 @@ namespace
 float
 rad (float deg)
 {
-    return deg * (M_PI / 180);
+    return deg * float(M_PI / 180);
```

Comment:
Up above, you used static_cast, but elsewhere, the constructor form. Any particular reason not to use one form everywhere?

Reply:
I'm not sure I was super consistent about this, in some places like this one the static cast is a bit too verbose and doesn't add much value. I'll try to do another pass soon to double-check.

I tend to prefer the static cast for cases involving a variable or a templated type since otherwise you can get into the ambiguous corners of the language (especially `T(foo)` which can mean a bunch of different things depending on context).

Reply:
In that case I think the choices you've made are consistent with clarity vs exactness. I commented one case at the top of the file where float(foo) would be concise and unambiguous.